pharus/interface.py

```diff
@@ -198,13 +200,16 @@ def _fetch_records(
                     elif attribute_info.type[0:7] == "decimal":
                         # Covert decimal to string
                         row.append(str(non_blobs_row[attribute_name]))
-                    else:
-                        # Normal attribute, just return value with .item to deal with numpy
-                        #   types
-                        if isinstance(non_blobs_row[attribute_name], np.generic):
-                            row.append((non_blobs_row[attribute_name].item()))
+                    # Normal attribute, just return value with .item to deal with numpy
+                    # types
+                    elif isinstance(non_blobs_row[attribute_name], np.generic):
+                        val = non_blobs_row[attribute_name].item()
+                        if isinstance(val, Number) and math.isnan(val):
+                            row.append(str(val))
```

Comment:
Lines 207-8 are the logical change in this PR. We catch `nan` values and cast to `str(nan)`.